frontend_tests/node_tests/compose.js

```diff
@@ -493,8 +507,8 @@ test_ui("update_fade", ({override}) => {
     initialize_handlers({override});
 
     const selector =
-        "#stream_message_recipient_stream,#stream_message_recipient_topic,#private_message_recipient";
-    const keyup_handler_func = $(selector).get_on_handler("keyup");
+        "#compose_select_stream_name,#stream_message_recipient_topic,#private_message_recipient";
+    const keyup_handler_func = $(selector).get_on_handler("change");
```

Comment:
I changed this to `change` since keyboard input isn't relevant for stream names changing anymore (now it's when an item in the dropdown is selected)

Reply:
That seems correct.